### PR TITLE
Tweak logout functionality

### PR DESCRIPTION
**Pull request by tsteer**

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -5,15 +5,37 @@ def log_in(user)
   end
 
   def current_user
-    @current_user ||= User.find_by(id: session[:user_id])
+    if(user_id = session[:user_id])
```
Comment (SebAshton) [inline app/helpers/sessions_helper.rb]: where does user_id come from?

Comment (SebAshton) [reply app/helpers/sessions_helper.rb]: Not a fan of assignments in `if` statements. Leads to undue cognitive overhead.

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -5,15 +5,37 @@ def log_in(user)
...
+      end
+    end
   end
 
   def logged_in?
     !current_user.nil?
   end
 
   def logout
+
```
Comment (SebAshton) [inline app/helpers/sessions_helper.rb]: Whitespace

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (tsteer) [reply app/helpers/sessions_helper.rb]: I'm assigning user_id, not comparing 😊

**Pull request merged**